routing/missioncontrol.go

```diff
@@ -377,28 +333,21 @@ func (m *MissionControl) GetHistorySnapshot() *MissionControlSnapshot {
 	defer m.Unlock()
 
 	log.Debugf("Requesting history snapshot from mission control: "+
-		"node_failure_count=%v, pair_result_count=%v",
-		len(m.lastNodeFailure), len(m.lastPairResult))
+		"pair_result_count=%v", len(m.lastPairResult))
 
-	nodes := make([]MissionControlNodeSnapshot, 0, len(m.lastNodeFailure))
-	for v, h := range m.lastNodeFailure {
-		otherProb := m.getPairProbability(v, route.Vertex{}, 0)
-
-		nodes = append(nodes, MissionControlNodeSnapshot{
-			Node:             v,
-			LastFail:         h,
-			OtherSuccessProb: otherProb,
-		})
-	}
+	nodes := make([]MissionControlNodeSnapshot, 0)
```

Comment:
Style nit: can instead just declare it (guessing you hit the "should pre allocate" directive in the liter):
```go
var nodes []MissionControlSnapshot
```

Reply:
If we _really_ want to pre-allocate though, we can add a method or function to compute the total size of the multi-dimensional map.

Reply:
Converted to var initializer. No, I don't necessarily want to pre-allocate :smile: